README.md

```diff
@@ -138,3 +138,45 @@ authorised_user:
 
 The email address has to exist in the users table of manage-courses-backend, but
 the password can be any non-secure local password you care to use.
+
+## Cypress
+
+1. Configurations file
+   1. A `./end-to-end-tests/config/example.json` is available as a basis to create `./end-to-end-tests/config/local.json`
+   1. Change the values where appropriate
+
+### Custom browser
+1. Download the appropriate version browser from [Chromium Downloads Tool](https://chromium.cypress.io/).
```

Comment:
I don't think we should include in the instructions, or at the very least make it clear that it's optional. If people have `Chrome` installed that'll be good enough.

On OSX, Cypress will search the standard places for binaries to run ... i.e. it picked up that I have `Chrome.app` and `Chromium.app` in `/Applications` and gives me the choice of which to use using the default mechanism. On Linux, doesn't it pick up `chrome` and `chromium` if it's in your path? If so, just tell people to install it with their package manager.

(btw, if we have to include these instructions, they'll be a little more complicated on MacOS because of code signing and the way things are run on MacOS)

Reply:
reworded